### ISSUE TITLE
Which files to move into scripts & plug-ins folder after macOS build?

### ISSUE THREAD

**Issue by aaronn** [iimachines/Maya2glTF#168]
Hi all, I've successfully built Maya2glTF for macOS, but it's not clear based on the instructions which files should be moved into /scripts/ and /plug-ins/ after completion.

Does anyone have a list of which files need to go into /scripts/ and which need to go into /plug-ins/? It seems like there's just a mix of files after the `make` command is run.

**Comment by Ziriax** [comment]
The `cmake install` script should deploy all files to the correct location, see [here](https://github.com/iimachines/Maya2glTF/blob/b418fbe7d47cf87cca6977461d9b296691de8d78/CMakeLists.txt#L191)

Maybe that isn't working anymore on the latest versions of OSX, I don't own a Mac anymore.

**Comment by psql** [comment]
yeah same issue here, looks like no files are deployed to the folder. Do they build to another directory first? Could be helpful if the install script put the `scripts` & `plug-ins` folders into a specified directory. 

Windows plugin works great, but would _love_ to get it working on MacOS! 🙏

**Comment by psql** [comment]
@aaronn it's possible you may have done the same thing I did (because it wasn't outlined in the instructions). After you run the `make` command, try `make install`. Works perfectly for me now 🤪
